sql/moz-fx-data-shared-prod/telemetry/retention_clients/view.sql

```diff
@@ -0,0 +1,127 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.retention_clients`
+AS
+WITH clients_last_seen AS (
+  SELECT
+    cls.submission_date,
+    cls.client_id,
+    cls.sample_id,
+    cls.normalized_channel,
+    mozfun.norm.os(cls.os) AS normalized_os,
+    COALESCE(
+      mozfun.norm.windows_version_info(cls.os, cls.os_version, cls.windows_build_number),
+      NULLIF(SPLIT(cls.normalized_os_version, ".")[SAFE_OFFSET(0)], "")
+    ) AS normalized_os_version,
+    cls.country,
+    cls.is_dau,
+    cls.is_wau,
+    cls.is_mau,
+    mozfun.bits28.retention(cls.days_seen_bits, cls.submission_date) AS retention_seen,
+    mozfun.bits28.retention(
+      cls.days_active_bits & cls.days_seen_bits,
+      cls.submission_date
+    ) AS retention_active,
+    cls.days_seen_bits,
+    cls.days_active_bits,
+  FROM
+    `moz-fx-data-shared-prod.telemetry.clients_last_seen_v2` cls
+  WHERE
+    cls.submission_date = @submission_date
+),
+new_profiles AS (
+  SELECT
+    client_id,
+    sample_id,
+    first_seen_date,
+    country,
+    locale,
+    app_version,
+    attribution_campaign,
+    attribution_content,
+    attribution_dlsource,
+    attribution_medium,
+    attribution_ua,
+    distribution_id,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    TRUE AS is_new_profile
+  FROM
+    `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v2`
```

Comment:
I think we'll want to filter this portion of the query given by date as well for performance.

Reply:
How far back do we want to go?